### PR TITLE
Animation variant

### PR DESCRIPTION
**Pull request by Thurst-ft**
Refactor to make the AB switch to just the part of the ui that is changing not the whole template.

Main diffence between this and the dark theme is I've ditched the multiple template route and added the changes as conditionals within one template. 

It is a bit more work but realised I'd been duplicating a lot fo the styles just to create a variant on one element. This way will be much cleaner. I think

### CODE REVIEW

File: src/lib/mapEventData.js [hunk 1]
```diff
@@ -1,7 +1,22 @@
 const oDate = require('o-date');
 
-module.exports = (theEvent) => {
+const staticAssets = 'https://www.ft.com/__assets/creatives/better-promo/';
```
Comment (ludrob) [inline src/lib/mapEventData.js]: ok for now, but we could move it to some sort of config or even better attach it the payload of the getEvent call (then we don't have anything hard-coded client side)

File: src/lib/mapEventData.js [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+
+	//AB test logic remove/refactor on test conclusion
+	let images = [];
+	let showVariant = false;
+
+	if(variant === 'variant') {
+		showVariant = true;
+		images.push(
+			theEvent.imageUrl,
+			`${staticAssets}event_break_out.jpg`,
```
Comment (ludrob) [inline src/lib/mapEventData.js]: ok for now, but we could move it to some sort of config or event attach it the payload of the getEvent call (then we don't have anything hard-coded client side)

**Comment by Thurst-ft** [comment]
Yup, still got some work to do around the animation and keeping the UI as true to the designs as possible. Without the tabs it actually looks quite different.

File: src/lib/mapEventData.js (hunk 2 above)
Comment (Thurst-ft) [reply src/lib/mapEventData.js]: Seems like a repeat of the comment above. Is that intentional? So the suggestion is move the `staticAssets` and the  'images'  array out of the client side?

Comment (ludrob) [reply src/lib/mapEventData.js]: Yes it is intentional. **It's not something we need to do right now**, as we are still in testing phase but let's say we keep this template with multiple images. We should probably move all references to images to the api layer and feed them to the app when we call the api. This way, if we ever need to change the image configs, we just need to amend the api layer config with no impact on next-article or the users' browsers

**Pull request merged**